dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - sample_id
+          - activity_segment
+          - normalized_app_name
+          - normalized_channel
+          - country
+          - city
+          - days_seen_bits
+          - days_since_first_seen
+          - days_since_seen
+          - device_model
+          - is_new_profile
+          - first_seen_date
+          - locale
+          - normalized_os
+          - durations
+          - normalized_os_version
+          - app_version
+          - os_version_major
+          - os_version_minor
+          - os_version_patch
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - activity_segment
+        expected_values:
+          - casual_user
+          - regular_user
+          - infrequent_user
+          - other
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_app_name
+        expected_values:
+          - Fenix
+          - "Fenix BrowserStack"
+          - "Firefox iOS"
+          - "Firefox iOS BrowserStack"
+          - "Firefox Desktop"
+          - "Focus iOS"
+          - "Focus Android"
+          - "Focus Android Glean"
+          - "Focus Android Glean BrowserStack"
```

Comment:
Good catch for the monitoring system already, thanks! "Focus Android Glean BrowserStack" must be filtered out from the view [mozdata.unified_metrics view](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql). I raised [PR 3360](https://github.com/mozilla/bigquery-etl/pull/3360) to correct it.